core/src/main/resources/org/apache/spark/ui/static/webui-dataTables.css

```diff
@@ -42,14 +42,6 @@ table.dataTable thead .sorting_desc_disabled::after {
   content: unset;
 }
 
-table.dataTable thead .sorting {
```

Comment:
Had to drop this because the newer version doesn't use up/down arrow icons so if we keep this double arrows appear...